sqflite/lib/src/connection.dart

```diff
@@ -0,0 +1,194 @@
+import 'dart:async';
+
+import 'package:jaguar_query/jaguar_query.dart';
+import 'package:jaguar_query_sqflite/composer.dart';
+import 'package:sqflite/sqflite.dart' as sqf;
+
+class SqfConn extends Connection<sqf.Database> {
+  sqf.Database _connection;
+
+  /// Connects to the database
+  static Future<SqfConn> open(String path, {int version, Logger logger}) async {
+    final connection = await sqf.openDatabase(path, version: version);
```

Comment:
Problem with this method is that openDatabase allow you much more that just path and version, I'm using it to manage migration because you can configure `onUpgrade`, `onDowngrade`, `onCreate`, `onOpen`, `onConfigure`, with your modifications we loose all of this

Reply:
While I was trying to make a migration (fallowing posgres changes) I was not able to make this.

In my app am doing migration out of jaguar. I am performing migration first, and make all preparation before using jaguar.

Can we make this merge now as it is, and try to do an optional update in the next migration phase.

As you said, there are lots of changes. The current master is totally unstable for sqflite and this merge can at least make it more stable.

Reply:
postgres and sqflite are really different, I don't think you should try to strictly follow what have been done as both are not working the same from the start.